apps/studio/src/schemas/collection.ts

```diff
@@ -66,6 +66,11 @@ export const createCollectionSchema = z.object({
   parentFolderId: z.number().optional(),
 })
 
+export const getCollectionTagsSchema = z.object({
+  resourceId: z.number().min(1),
+  siteId: z.number().min(1),
+})
```

Comment:
nit: think it's about time we centralise this, feels like this zod object combination is used very frequently